.github/CONTRIBUTING.md

```diff
@@ -275,7 +275,7 @@ Where we can, we use their API to get the data, so that we have completely repro
 
 On the [ONS Open Geography portal](https://geoportal.statistics.gov.uk/), you will usually be looking for data published as a feature or feature layer, as these are the ones made available via the API connection. You'll be able to preview the data in the browser and do basic searching / filtering on the table if you want to visualise it. Any feature data should have an option somewhere for 'I want to use this data' (or something similar if they update their website design) where you can get to an API explorer that allows you to run a basic query in the browser. In here you can usually find the dataset_id and also the parameters you want to use to get the data you need.
 
-We have a `get_ons_api_data()` function that acts as a wrapper to the ONS API, it does things like converting readable parameters into a query string and also handles batching and multiple requests if needed, so you get all of the data in one nice neat data frame (there's a limit on the rows per single query for the API).
+We have a `get_ons_api_data()` function that acts as a wrapper to the ONS API, it does things like converting readable parameters into a query string and also handles batching and multiple requests if needed, so you get all of the data in one nice neat data frame (there's a limit on the rows per single query for the API). If you're looking to expand on this function at all, you should first check if the [boundr package](https://github.com/francisbarton/boundr) does what you need, as that gives a number of methods for extracting data from the portal as well.
```

Comment:
When would you suggest using our function vs using boundr?

Reply:
Generally use ours first as it's built for our needs in the package. Then check boundr if ours doesn't do what you need, and then if neither do make a suggestion on here if it's DfE specific or on boundr if more general / others might benefit too

Updated to put that into the text